README.md

```diff
@@ -1,2 +1,12 @@
 # ZnanyProgramista
 Projekt końcowy - ZnanyProgramista
+
+żeby zacąć 
+sprawdź requirements.txt (nowa bimlioteka)
+python manage.py makemigrations accounts
+python manage.py migrate
+python manage.py createsuperuser
+python manage.py runserver 7777
```

Comment:
proponowałbym zrobić README porządni na końcu, bo teraz to co chwilę trzebaby coś tu zmieniać, no i to trzeba zrobić dużo obszerniejsze, wg mnie najlpiej po angielsku, korzystać z markdown itd itp